samza-core/src/main/java/org/apache/samza/system/SystemAdmins.java

```diff
@@ -60,4 +64,8 @@ public SystemAdmin getSystemAdmin(String systemName) {
     }
     return systemAdminMap.get(systemName);
   }
+
+  public Set<String> getAdminKeys() {
```

Comment:
Maybe name the method to what the keys represent? E.g. getSystemNames?

Reply:
Resolved